src/dswx_sar/refine_with_bimodality.py

```diff
@@ -1040,15 +1101,13 @@ def remove_false_water_bimodality_parallel(water_mask_path,
     else:
         merged_removed_false_water_path = remove_false_water_path_set[0]
 
-    bimodality_total = dswx_sar_util.read_geotiff(merged_removed_false_water_path)
-
-    return bimodality_total
+    return merged_removed_false_water_path
 
 
 def fill_gap_water_bimodality_parallel(
         bright_water_path,
         pol_list,
-        threshold = [0.7, 1.5],
+        threshold=[0.7, 1.5],
```

Comment:
It is generally not a good idea to provide a mutable variable as a default value. More information:
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/dangerous-default-value.html

Reply:
Thanks for pointing out. But these values are given from the runconfing, so we can change the value later. 
https://github.com/opera-adt/DSWX-SAR/blob/967bce79a36ba86e606d673b3c7733575fae0512/src/dswx_sar/refine_with_bimodality.py#L1367-L1376

Reply:
I agree with @seongsujeong. According to the link, the safe way would be the following:
```python
def fill_gap_water_bimodality_parallel(
        bright_water_path,
        pol_list,
        threshold=None,
        ...)
        # after docstring
        if threshold is None:
            threshold = [0.7, 1.5]
```
Can you explain what you mean by "change the value later"? Either way a default is applied and the default values in the code can be changed.

Reply:
Ah. What I meant was that we can control these parameters from the runconfig. They are given from the input runconfig.